src/pages/Cart/Cart.jsx

```diff
@@ -1,42 +1,31 @@
 import React from "react";
-import {CartCard} from "../../components/component";
+import { CartCard, EmptyProd, CartCheckOut } from "../../components/component";
+import { useCart } from "../../context/cartContext";
 
 export default function Cart() {
+  const { state } = useCart();
+  const Cart = state.CartItems;
+
+  const CartMapping = () => {
+    Cart.map((item) => {
```

Comment:
you can directly give state.CartItems array here  instead of Cart.
because product is been added to cart but not been display. But Billing component is working.

Reply:
I will fiz it